packages/vaadin-charts/src/vaadin-chart.js

```diff
@@ -226,7 +226,7 @@ export const deepMerge = function deepMerge(target, source) {
  * @fires {CustomEvent} point-remove - Fired when the point is removed from the series.
  * @fires {CustomEvent} point-select -Fired when the point is selected either programmatically or by clicking on the point.
  * @fires {CustomEvent} point-unselect - Fired when the point is unselected either programmatically or by clicking on the point.
- * @fires {CustomEvent} point-update - Fired when the point is updated programmatically through `.update()` method.
```

Comment:
The examples above still use the deprecated API `this.$.mychart.update` (should be fixed in the d.ts file also). Other than that, lgtm

Reply:
Good catch, thanks! Fixed.